lib/benchee/project/benchmark.ex

```diff
@@ -0,0 +1,175 @@
+defmodule Benchee.Project.Benchmark do
+  @moduledoc """
+  Module with macro for defining project-wide benchmarks.
+  """
+  require Logger
+
+  def benchee_scenario_attr, do: :benchee_cases
```

Comment:
still don't fully understand why we need this (I know it's used in the bottom but I don't completely understand it)

Reply:
Ok I guess this is the module attribute where we basically compile whatever we compile as benchmarks in here?

I'd name it `benchmarks` though as `cases` isn't something we have in our lingo right now.

Reply:
As to "why": simply a matter of taste: compiler won't warn you on atom typo, but will on function name typo :)

Note about the name is taken into account :)